bot/code_review_bot/workflow.py

```diff
@@ -103,6 +102,47 @@ def run(self, revision):
 
         return issues
 
+    def ingest_autoland(self, revision):
+        """
+        Simpler workflow to ingest autoland revision
+        """
+        assert revision.repository == REPO_AUTOLAND, "Need an autoland revision"
+        logger.info(
+            "Starting autoland ingestion",
+            revision=revision.id,
+            bugzilla=revision.bugzilla_id,
+            title=revision.title,
+            mercurial_revision=revision.mercurial_revision,
+        )
+
+        assert (
+            self.backend_api.enabled
+        ), "Backend storage is disabled, no autoland ingestion possible"
+
+        # Find potential issues in the autoland task group
+        # There should be no issues, so the build is OK
+        tasks = self.queue_service.listTaskGroup(settings.autoland_group_id)
+        issues = []
+        for task_status in tasks["tasks"]:
+            task = self.build_task(task_status, skip_unknown=True)
+
+            if task is None:
+                continue
+
+            artifacts = task.load_artifacts(self.queue_service)
+            if artifacts is not None:
+                task_issues = task.parse_issues(artifacts, revision)
+                logger.info(
+                    "Found {} issues".format(len(task_issues)),
+                    task=task.name,
+                    id=task.id,
+                )
+                issues += task_issues
+
+        # Store the revision & diff in the backend
+        # TODO: publish state (issues or not issues)
```

Comment:
Isn't this goint to be available from the backend?

Reply:
Yeap:
- no issues on autoland revision :heavy_check_mark: clean before state
- some issues on autoland revision :negative_squared_cross_mark: we have to compare the after issues with these ones